### PR TITLE
Google oauth write up

### PR DESCRIPTION
**Pull request by rdodev**
Adds instructions, examples and schreenshots how to add Google as an OAuth provider.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,12 +15,14 @@ a collection of utilities that can be used to enable authn/authz flows for a kub
...
-[auth0 example](examples/auth0-gangway-example.yaml)
+* [auth0 example](examples/auth0-gangway-example.yaml)
+
+* [Google OAuth example](#google-oauth-config)
 
 
 ### Creating Config Secret
 
-The [gangway config](#gangway-config) should be stored as a secret in Kubernetes becaue it contains sensitive information. Create a gangway config based on the example, assuming you named the file `gangway-config.yaml` run the following command: `kubectl create secret generic gangway --from-file=gangway.yaml=gangway-config.yaml`
+The [gangway config](#gangway-config) should be stored as a secret in Kubernetes becaue it contains sensitive information. Create a gangway config based on the example, assuming you named the file `gangway-config.yaml` run the following command: `kubectl create secret generic gangway --from-file=gangway.yaml`
```
Comment (craigtracey) [inline README.md]: Is there a reason why we need to change this?

File: examples/gangway-google-config.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Your Cluster Name. There's no strict mapping, so it can be anything
```
Comment (craigtracey) [inline examples/gangway-google-config.yaml]: In 1.0.0 these fields went from snake case to camel case, in order to be more consistent with Kube.

File: examples/google-gangway-example.yaml [hunk 3]
```diff
@@ -0,0 +1,82 @@
+
```
Comment (craigtracey) [inline examples/google-gangway-example.yaml]: Is there a reason why this should be unique to Google?

Comment (rdodev) [reply examples/google-gangway-example.yaml]: No, it isn't. I must've added it by accident. Will remove.

File: examples/gangway-google-config.yaml (hunk 2 above)
Comment (rdodev) [reply examples/gangway-google-config.yaml]: to be clear, you mean `ClusterName`, `AuthorizeUrl` as opposed to using underscores?

File: README.md (hunk 1 above)
Comment (rdodev) [reply README.md]: thought it was a merge error given the two `=`

File: examples/gangway-google-config.yaml (hunk 2 above)
Comment (craigtracey) [reply examples/gangway-google-config.yaml]: clusterName and authorizeURL. The readme was updated with the list.

File: README.md (hunk 1 above)
Comment (craigtracey) [reply README.md]: This is how you can specify the file name as exposed by the Secret.

File: examples/gangway-google-config.yaml (hunk 2 above)
Comment (rdodev) [reply examples/gangway-google-config.yaml]: Thanks.  Will update.

**Comment by craigtracey** [comment]
Thanks for this update! Once we get these comments addressed, please squash the commits with a descriptive commit message and signoff.

File: README.md (hunk 1 above)
Comment (rdodev) [reply README.md]: Ah gotcha wasn't aware you could specify key-value through command line like that. Will restore to original value.

**Comment by rdodev** [comment]
Requested changes have been incorporated @craigtracey

**Comment by rdodev** [comment]
Now it's ready @craigtracey

**Comment by rdodev** [comment]
@craigtracey can we get this merged?

**Pull request merged**